stubs/passlib/passlib/apache.pyi

```diff
@@ -1,59 +1,100 @@
-from _typeshed import Incomplete
-from typing import Any
+from typing_extensions import Self
+
+from .context import CryptContext
+from .hash import htdigest
 
 class _CommonFile:
-    encoding: Any
-    return_unicode: Any
+    encoding: str | None
+    return_unicode: bool | None
```

Comment:
While those are initialized to `None` in the class, they are immediately set in `__init__`, so they should never be `None` in instances:

```suggestion
    encoding: str
    return_unicode: bool
```

Reply:
> While those are initialized to `None` in the class, they are immediately set in `__init__`, so they should never be `None` in instances:

Unfortunately that means we need a stubtest_allowlist entry, but it is what it is.